### ISSUE TITLE
[Question/Support] How to setup AdvancedBan on an offline mode server?

### ISSUE THREAD

**Issue by itaquito** [DevLeoko/AdvancedBan#424]
Hello!

We currently have AdvancedBan working on our Bungee server. Since the UUIDs we use are offline, AB can't find the UUID of a player when he is disconnected. 
Is there a way to store the UUIDs for AdvancedBan?
Or, can all the punishments be name-based and not UUID-based?

Also, thanks for the recent updates. I think all the community really appreciates them :3
Thanks!

**Comment by DevLeoko** [comment]
Yes, you can simply disable the UUID-Fetcher in the config, which will make AdvancedBan use names instead of UUIDs.
Though be aware that old punishments won't work anymore after switching.
